src/angular-google-analytics.js

```diff
@@ -187,13 +192,28 @@
         };
 
         var _ga = function() {
+
           if (!$window.ga) {
             that._log('warn', 'ga function not set on window');
           }
           if (logAllCalls === true) {
             that._log.apply(that, arguments);
           }
-          $window.ga.apply(null, arguments);
+          // If multiple trackers && arguments valid && command supports multiple trackers
+          // Temporary check if command contains '.' until all functions are switched
+          if (angular.isArray(accountId) && arguments && arguments.length > 1 && arguments[0].indexOf('.') < 0 &&
+              trackingSupportingMultipleTrackers.indexOf(arguments[0]) >= 0) {
+            var args = Array.prototype.slice.call(arguments);
+            // Create command for each tracker an apply
+            accountId.forEach(function (trackerObj) {
+              var command = trackerObj.name + '.' + args[0];
+              args[0] = command;
+              $window.ga.apply(null, args);
```

Comment:
Alternatively, though it is a micro-optimization, you could add a statement at line 209 to set ```args[0] = arguments[0]``` so the first index is reset in the args array on each iteration.